src/models/OtherReason.schema.ts

```diff
@@ -0,0 +1,25 @@
+import * as Joi from '@hapi/joi';
+
+const titleErrorMessage = 'You must give your reason a title';
+const descriptionErrorMessage = 'You must give us more information';
+
+export const schema = Joi.object({
+    title: Joi.string()
+        .required()
```

Comment:
Is the required() check necessary? I would assume string.empty message would handle the case of an empty field